qutebrowser/config/configdata.yml

```diff
@@ -1445,6 +1445,12 @@ url.incdec_segments:
   desc: URL segments where `:navigate increment/decrement` will search for
     a number.
 
+url.open_base_url:
+  type: Bool
+  default: false
+  desc: Invoking `:open {shortcut}` (without argument), where {shortcut} is a search engine shortcut
+    will open the base url of the shortcut instead of using the default search engine.
```

Comment:
Would you mind rephrasing this to [imperative mood](https://en.wikipedia.org/wiki/Imperative_mood) to better match existing descriptions of other boolean options?
See https://github.com/qutebrowser/qutebrowser/pull/3181 for the whole discussion and https://github.com/qutebrowser/qutebrowser/commit/494aceec451e307b0227b0fdd0f86d419dbc11d1 for specific examples.